.github/workflows/cont_integration.yml

```diff
@@ -172,3 +172,6 @@ jobs:
       - name: Build
         working-directory: example-crates/${{ matrix.example-dir }}
         run: cargo build
+      - name: Test
+        working-directory: example-crates/${{ matrix.example-dir }}
+        run: cargo test
```

Comment:
Just noting I don't think there are any tests here to run.

Reply:
Oh, yes. Thank you!